hezar/metrics/__init__.py

```diff
@@ -1,7 +1,7 @@
-from .f1 import F1, F1Config
 from .accuracy import Accuracy, AccuracyConfig
 from .bleu import BLEU, BLEUConfig
 from .cer import CER, CERConfig
+from .f1 import F1, F1Config
 from .metric import Metric
```

Comment:
@HamedBabaei Forgot to mention this one. Metric must be on top.

Reply:
done!